tests/conftest.py

```diff
@@ -38,7 +38,8 @@ def pytest_sessionstart():
     airflow.models.Variable.set("AZ_BRANCH", "AZ_BRANCH")
     airflow.models.Variable.set("AZ_PROD_BRANCH", "AZ_BRANCH")
     airflow.models.Variable.set("AZ_QA_BRANCH", "AZ_BRANCH")
-    airflow.models.Variable.set("AZ_CONFIGSET", "tul_cob-az-1")
+    airflow.models.Variable.set("AZ_STAGE_BRANCH", "AZ_BRANCH")
```

Comment:
Stage should be using whatever is the prod branch.  So I'm a little confused by this change.

Reply:
Stage variables shouldn't exist at all anymore.  I'm not sure why those are still there...
Let me do some checking.  Something's not right.